seacrowd/sea_datasets/thai_romanization/thai_romanization.py

```diff
@@ -0,0 +1,190 @@
+import os
+from pathlib import Path
+from typing import Dict, List, Tuple
+
+import datasets
+import pandas as pd
+
+from seacrowd.utils import schemas
+from seacrowd.utils.configs import SEACrowdConfig
+from seacrowd.utils.constants import Tasks, Licenses
+
+# There are no citation available for this dataset.
+_CITATION = """\
+@article{,
+  author    = {},
+  title     = {},
+  journal   = {},
+  volume    = {},
+  year      = {},
+  url       = {},
+  doi       = {},
+  biburl    = {},
+  bibsource = {}
+}
+"""
+
+_DATASETNAME = "thai_romanization"
+
+_DESCRIPTION = """
+The Thai Romanization dataset contains 648,241 Thai words \
+    that were transliterated into English, making Thai \
+    pronounciation easier for non-native Thai speakers. \
+    This is a valuable dataset for Thai language learners \
+    and researchers working on Thai language processing task. \
+    Each word in the Thai Romanization dataset is paired with \
+    its English phonetic representation, enabling accurate \
+    pronunciation guidance. This facilitates the learning and \
+    practice of Thai pronunciation for individuals who may not \
+    be familiar with the Thai script. The dataset aids in improving \
+    the accessibility and usability of Thai language resources, \
+    supporting applications such as speech recognition, text-to-speech \
+    synthesis, and machine translation. It enables the development of \
+    Thai language tools that can benefit Thai learners, tourists, \
+    and those interested in Thai culture and language.
+"""
+
+_HOMEPAGE = "https://www.kaggle.com/datasets/wannaphong/thai-romanization/data"
+
+_LANGUAGES = ["tha"]
+
+_LICENSE = Licenses.CC_BY_SA_3_0.value
+
+_LOCAL = False
+
+_URLS = {_DATASETNAME: "https://raw.githubusercontent.com/wannaphong/thai-romanization/master/dataset/data.csv"}
+
+_SUPPORTED_TASKS = [Tasks.TRANSLITERATION]
+
+_SOURCE_VERSION = "1.0.0"
+
+_SEACROWD_VERSION = "1.0.0"
+
+
+class ThaiRomanizationDataset(datasets.GeneratorBasedBuilder):
+    """
+    The Thai Romanization dataset contains 648,241 Thai words \
+    that were transliterated into English, making Thai \
+    pronounciation easier for non-native Thai speakers. \
+    This is a valuable dataset for Thai language learners \
+    and researchers working on Thai language processing task. \
+    Each word in the Thai Romanization dataset is paired with \
+    its English phonetic representation, enabling accurate \
+    pronunciation guidance. This facilitates the learning and \
+    practice of Thai pronunciation for individuals who may not \
+    be familiar with the Thai script. The dataset aids in improving \
+    the accessibility and usability of Thai language resources, \
+    supporting applications such as speech recognition, text-to-speech \
+    synthesis, and machine translation. It enables the development of \
+    Thai language tools that can benefit Thai learners, tourists, \
+    and those interested in Thai culture and language.
```

Comment:
I think it's fine to have the long description in the `_DESCRIPTION` and the docstring here should just describe the dataloader

```suggestion
    Thai Romanization dataloader from Kaggle (Phong et al., 2018)
```

Reply:
Solved on https://github.com/SEACrowd/seacrowd-datahub/pull/645/commits/e1500c082185807f49cbf01bcdde7eab99578869 commit, Sir.